packages/core/src/base/staticClient.ts

```diff
@@ -6,16 +6,23 @@
  */
 import { default as fetch, Response, RequestInit } from "make-fetch-happen";
 import _ from "lodash";
+import fetchToCurl from "fetch-to-curl";
 
 import DefaultCache = require("make-fetch-happen/cache");
 export { DefaultCache, Response };
 
 import { Resource } from "./resource";
 import { BaseClient } from "./client";
 import { sdkLogger } from "./sdkLogger";
-import fetchToCurl from "fetch-to-curl";
 
 const CONTENT_TYPE = "application/json";
+//sensitive fields in request body that need to be masked while logging.
+export const SENSITIVE_FIELDS: ReadonlyArray<string> = [
+  "password".toLowerCase(),
+  "newPassword".toLowerCase(),
+  "currentPassword".toLowerCase()
```

Comment:
We should add the following:
- Credit card field(s)
- Address field(s)
- SSN field(s)
- DOB field(s)

We should also expose this to customers so that they can add their own

Reply:
I thought about exposing but was not sure of the best way as we do not have global config.